rules/integrations/cloud_defend/privilege_escalation_potential_container_escape_via_modified_notify_on_release_file.toml

```diff
@@ -0,0 +1,60 @@
+[metadata]
+creation_date = "2023/10/26"
+integration = ["cloud_defend"]
+maturity = "production"
+min_stack_comments = "New Integration: Cloud Defend"
+min_stack_version = "8.8.0"
+updated_date = "2023/10/26"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects modification of the cgroup notify_on_release file from inside a container. When the notify_on_release
+flag is enabled (1) in a cgroup, then whenever the last task in the cgroup exits or attaches to another cgroup, the
+command specified in the release_agent file is run and invoked from the host. A privileged container with SYS_ADMIN
+capabilities, enables a threat actor to mount a cgroup directory and modify the notify_on_release flag in order to take
+advantage of this feature, which could be used for further privilege escalation and container escapes to the host
+machine.
+"""
+from = "now-6m"
+index = ["logs-cloud_defend*"]
+interval = "5m"
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Container Escape via Modified notify_on_release File"
+references = [
+    "https://blog.trailofbits.com/2019/07/19/understanding-docker-container-escapes/",
+    "https://sysdig.com/blog/detecting-mitigating-cve-2022-0492-sysdig/",
+]
+risk_score = 73
+rule_id = "ef65e82c-d8b4-4895-9824-5f6bc6166804"
+severity = "high"
+tags = [
+    "Data Source: Elastic Defend for Containers",
+    "Domain: Container",
+    "OS: Linux",
+    "Use Case: Threat Detection",
+    "Tactic: Privilege Escalation",
+]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+file where event.module == "cloud_defend" and event.action=="open" and 
+event.type== "change" and file.name: "*notify_on_release"
```

Comment:
I'd suggest the same for this one, to use the `file.path: "/sys/fs/cgroup/*notify_on_release` if possible.

Reply:
I agree here better if possible to state the full path to make the query more efficient.

Reply:
@Aegrah @DefSecSentinel for this attack I was able to create new user and cgroup namespaces in a /tmp directory. 
https://sysdig.com/blog/detecting-mitigating-cve-2022-0492-sysdig/ 
Would you still suggest using file.path with a wildcard to account for unconventional locations? 
![image](https://github.com/elastic/detection-rules/assets/59296946/e096b235-b7ff-4a32-9c59-9f7f355d88c8)

Reply:
I'm going to remove the leading wildcard as I did with the release_agent rule, thank you!